square/shwap/row.go

```diff
@@ -85,17 +95,14 @@ func (r Row) IsEmpty() bool {
 
 // Verify checks if the row's shares match the expected number from the root data and validates
 // the side of the row.
-func (r Row) Verify(roots *share.AxisRoots, idx int) error {
+func (r Row) Verify(roots *square.AxisRoots, idx int) error {
 	if len(r.halfShares) == 0 {
 		return fmt.Errorf("empty half row")
 	}
 	expectedShares := len(roots.RowRoots) / 2
 	if len(r.halfShares) != expectedShares {
 		return fmt.Errorf("shares size doesn't match root size: %d != %d", len(r.halfShares), expectedShares)
 	}
-	if err := ValidateShares(r.halfShares); err != nil {
-		return fmt.Errorf("invalid shares: %w", err)
-	}
```

Comment:
Why this get dropped?

Reply:
https://github.com/celestiaorg/go-square/blob/a446cee3ad5fd5be5e59dc9a027bc35b4f6100cb/share/share.go#L23